src/app/components/cart/cart.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { Card } from '../products/models/card.interface';
+import { PaginationService } from '../products/core/pagination/pagination.service';
+import { CartService } from '../products/core/cart/cart.service';
+
+@Component({
+  selector: 'app-cart',
+  templateUrl: './cart.component.html',
+  styleUrls: ['./cart.component.scss']
+})
+export class CartComponent implements OnInit {
+
+  
+  cart:Card[] = []
+  numberOfPages = 1
+  currentPageProducts:Card[] = []
+  constructor(private CartService: CartService, private PaginationService: PaginationService) { }
```

Comment:
naming of constructor parameters usually done in lowercase.
`private cartService: CartService, private paginationService: PaginationService`